### PR TITLE
add doc for identityStoreId

### PR DESCRIPTION
**Pull request by jennaxue-okta**
## Description:
- **What's changed?** add add for a new property of Apps API: identityStoreId 
- **Is this PR related to a Monolith release?** YES 2023.05.0

### Resolves:

* [OKTA-603693](https://oktainc.atlassian.net/browse/OKTA-603693)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/fe56b249c4d9bd06fbf481a0ab575b3c18d0a614/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f6ef1f3f-fe27-4f65-b653-3a0f767d3e47) | 65  | 67  | 69  | 71  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.627437ce-7ead-4941-938e-91f9230cafa9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7861]:
```suggestion
| identityStoreId       | Specify a valid identity store application ID (from the same org) if this application supports an identity store.                    | String  | TRUE    | NULL   |           |
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
 
 > **Note:** You can't currently manage app provisioning settings via the API. Use the administrator UI.
 
+###### Identity Store Id
+An additional `identityStoreId` can be passed within the `settings` object if application supports identity store. `identityStoreId` should be a valid identity store application id in the same Org.
+
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7866]: Suggest to remove this section and put the `identityStoreId` parameter in the **App Settings** parameter table.

Question about the `identityStoreId` value: is it an `appId` of another app created in the same org representing the external identity store app? Or is it a GUID from an external source?

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Hi Jenna, I suggest we move the parameter description to the `settings` table. And I have another question regarding the value (I might have to change the RN blurb). Thank you!

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 2 above)
Comment (jennaxue-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7866]: it's appId of another app created in the same org.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
 
 > **Note:** You can't currently manage app provisioning settings via the API. Use the administrator UI.
 
+###### Identity Store Id
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7864]:
```suggestion
###### Identity Store ID
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
...
+An additional `identityStoreId` can be passed within the `settings` object if application supports identity store. `identityStoreId` should be a valid identity store application id in the same Org.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7865]:
```suggestion
You can set an additional `identityStoreId` string parameter within the `settings` object if your application supports an identity store. The `identityStoreId` value must be a valid identity store application `id`. This identity store application must be created in the same org as your application.
```

**Review by vanngo-okta — COMMENTED** [review]
1Please review the suggested description. Thanks

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 3 above)
Comment (jennaxue-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done in https://github.com/okta/okta-developer-docs/pull/4114/commits/e00c00ad00dbc75617d9ec9517a747d2a555dd74

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 4 above)
Comment (jennaxue-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done in https://github.com/okta/okta-developer-docs/pull/4114/commits/e00c00ad00dbc75617d9ec9517a747d2a555dd74

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/apps/index.md](https://github.com/okta/okta-developer-docs/blob/e00c00ad00dbc75617d9ec9517a747d2a555dd74/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fapps%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/db26e30e-5412-4148-9e29-1de2021d8772) | 65  | 68  | 69  | 71  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.8cfffe8a-5d8c-4ebc-a578-8f75785fb273?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Review by ramyamudduluri-okta — APPROVED** [review]

**Pull request merged**